packages/react-scripts/package.json

```diff
@@ -76,12 +76,13 @@
     "terser-webpack-plugin": "1.2.3",
     "ts-pnp": "1.1.2",
     "url-loader": "1.1.2",
-    "webpack": "4.35.0",
+    "webpack": "4.29.6",
```

Comment:
@trayldev instead of making this change, I'd like to see us branch from the `3.0.1` tag and rebase all our changes against those commits. That way our Git history is nice and clean and it's easier to apply. Are you comfortable with Git rebasing or would it be easier for me to handle it? Once we've done that, then I think this will be all set for the future